### DISCUSSION TITLE
✅ 🔗 Open Portfolio links in new tab (Duplicate)

### DISCUSSION THREAD

**Discussion by trilobutt** [elementor/elementor#14306] [Feature Request]
It would be great if we can have a button to open Portfolio links in a new tab, similar to what the Post module has. I hacked this in the PHP, but it will go away every time I update the plugin.

Thank you!

**Comment by nicholaszein** [comment]
Hello,

this is now possible to achieve using the new Loop Grid widget to build your layout for your portfolio by adding an image with a link that opens in a new tab.

To learn more about the Loop Grid widget check [this article](https://elementor.com/help/loop-grid/).

We have no plans to further develop the Portfolio widget, as the Loop Grid gives you much flexibility.

I'm closing this request as solved.

Kind regards
